### PR TITLE
feat: add mqtt internal url in tests

### LINKED ISSUE #238: Adding a second Test configuration when using FIWARE components inside Docker
There is currently the problem that not all tests run through when working inside a local Docker environment. This occurs in most cases with subscriptions, as the containers cannot communicate with each other via `localhost`. For example, you cannot send an MQTT subscription to the URL `mqtt://127.0.0.1:1883`, but must send it to `mqtt://<mosquitto container hostname>:1883` in the case of Docker. There is no bug, but it would be useful if there was a second test config file for such URLs (of each FIWARE component).

### PR DESCRIPTION
**Pull request by djs0109**
close #238

### CODE REVIEW

**Review by RCX112 — COMMENTED** [review]
That is exactly what I mean with the MQTT subscriptions. More general, in the Issue #238 we would also need an internal URL for other FIWARE components such as the OCB, for example in https://github.com/RWTH-EBC/FiLiP/blob/a559c26fdc97a1835540f01afda66a9ddc6fb464/tests/clients/test_ngsi_v2_timeseries.py#L97

**Comment by RCX112** [comment]
@djs0109 If I checked everything correctly, the only missing thing is an internal cb_url for the timeseries test function which I mentioned on the top:

https://github.com/RWTH-EBC/FiLiP/blob/a559c26fdc97a1835540f01afda66a9ddc6fb464/tests/clients/test_ngsi_v2_timeseries.py#L97

File: tests/clients/test_ngsi_v2_cb.py [hunk 1]
```diff
@@ -533,6 +534,7 @@ def on_disconnect(client, userdata, reasonCode, properties=None):
                 cb_url=settings.CB_URL)
     def test_notification(self):
         mqtt_sub_url = settings.MQTT_BROKER_URL
```
[the remark this answers is not in the document]
Comment (djs0109) [reply tests/clients/test_ngsi_v2_cb.py]: It has been changed now

**Comment by djs0109** [comment]
@RCX112  I would rather not implement it, because this API has been deprecated in QL1.0.0 (see here https://github.com/orchestracities/ngsi-timeseries-api/pull/493). I will open up another issue to test the capability with QL 1.0.0 and remove this test

**Review by RCX112 — APPROVED** [review]
@djs0109 Looks good, we can merge :)

**Pull request merged**